plugin/trino-http-event-listener/src/test/java/io/trino/plugin/httpquery/TestHttpEventListenerConfig.java

```diff
@@ -13,25 +13,76 @@
  */
 package io.trino.plugin.httpquery;
 
+import io.airlift.configuration.validation.FileExists;
 import io.airlift.units.Duration;
 import org.junit.jupiter.api.Test;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
 import static io.airlift.configuration.testing.ConfigAssertions.assertFullMapping;
 import static io.airlift.configuration.testing.ConfigAssertions.assertRecordedDefaults;
 import static io.airlift.configuration.testing.ConfigAssertions.recordDefaults;
+import static io.airlift.testing.ValidationAssertions.assertFailsValidation;
+import static org.assertj.core.api.Assertions.assertThat;
 
 final class TestHttpEventListenerConfig
 {
     @Test
-    void testDefaults()
+    void testGetHttpHeadersLoadsFromConfigFile()
             throws Exception
+    {
+        // Create a temp file with header values in properties format, including single and double quotes
+        Path tempFile = Files.createTempFile("headers", ".properties");
+        String fileContent = "Authorization=Trust Me\nCache-Control=no-cache\nCustom-Header='single-quoted'\nAnother-Header=\"double-quoted\"\n";
+        Files.writeString(tempFile, fileContent);
+
+        HttpEventListenerConfig config = new HttpEventListenerConfig();
+        config.setHttpHeadersConfigFile(tempFile.toFile());
+
+        Map<String, String> headers = config.getHttpHeaders();
+        assertThat(headers.size()).isEqualTo(4);
+        assertThat(headers.get("Authorization")).isEqualTo("Trust Me");
+        assertThat(headers.get("Cache-Control")).isEqualTo("no-cache");
+        assertThat(headers.get("Custom-Header")).isEqualTo("'single-quoted'");
+        assertThat(headers.get("Another-Header")).isEqualTo("\"double-quoted\"");
```

Comment:
These assertions don't return helpful info when failed. Please change to `assertThat(headers).contains...` instead.